services/app-api/src/postgres/contractAndRates/findContractWithHistory.test.ts

```diff
@@ -299,76 +299,77 @@ describe.skip('findContractWithHistory with full contract and rate history', ()
+        const submissionsInTimeOrder =
+            resultingContract.packageSubmissions.reverse()
 
         console.info(
             'ALL First REvisions: ',
-            JSON.stringify(revisionsInTimeOrder, null, '  ')
+            JSON.stringify(submissionsInTimeOrder, null, '  ')
         )
 
         // Each Revision needs a Reason, one of the contracts or revisions associated with it should have changed and why.
 
-        expect(revisionsInTimeOrder).toHaveLength(8)
-        expect(revisionsInTimeOrder[0].rateRevisions).toHaveLength(0)
-        expect(revisionsInTimeOrder[0].unlockInfo).toBeUndefined()
-        expect(revisionsInTimeOrder[0].submitInfo?.updatedReason).toBe(
+        expect(submissionsInTimeOrder).toHaveLength(8)
+        expect(submissionsInTimeOrder[0].rateRevisions).toHaveLength(0)
+        // expect(submissionsInTimeOrder[0].unlockInfo).toBeUndefined()
+        expect(submissionsInTimeOrder[0].submitInfo?.updatedReason).toBe(
             'initial submit'
         )
 
-        expect(revisionsInTimeOrder[1].rateRevisions).toHaveLength(1)
-        expect(revisionsInTimeOrder[1].unlockInfo).toBeUndefined()
-        expect(revisionsInTimeOrder[1].submitInfo?.updatedReason).toBe(
+        expect(submissionsInTimeOrder[1].rateRevisions).toHaveLength(1)
+        // expect(submissionsInTimeOrder[1].unlockInfo).toBeUndefined()
+        expect(submissionsInTimeOrder[1].submitInfo?.updatedReason).toBe(
             'Rate Submit'
         )
 
-        expect(revisionsInTimeOrder[2].rateRevisions).toHaveLength(2)
-        expect(revisionsInTimeOrder[2].unlockInfo).toBeUndefined()
-        expect(revisionsInTimeOrder[2].submitInfo?.updatedReason).toBe(
+        expect(submissionsInTimeOrder[2].rateRevisions).toHaveLength(2)
+        // expect(submissionsInTimeOrder[2].unlockInfo).toBeUndefined()
+        expect(submissionsInTimeOrder[2].submitInfo?.updatedReason).toBe(
             'RateSubmit 2'
         )
 
-        expect(revisionsInTimeOrder[3].rateRevisions).toHaveLength(3)
-        expect(revisionsInTimeOrder[3].unlockInfo).toBeUndefined()
-        expect(revisionsInTimeOrder[3].submitInfo?.updatedReason).toBe(
+        expect(submissionsInTimeOrder[3].rateRevisions).toHaveLength(3)
+        // expect(submissionsInTimeOrder[3].unlockInfo).toBeUndefined()
+        expect(submissionsInTimeOrder[3].submitInfo?.updatedReason).toBe(
             '3.0 create'
         )
 
-        expect(revisionsInTimeOrder[4].rateRevisions).toHaveLength(2)
-        expect(revisionsInTimeOrder[4].unlockInfo?.updatedReason).toBe(
-            'unlock for 2.1 remove'
-        )
-        expect(revisionsInTimeOrder[4].unlockInfo?.updatedBy).toBe(
-            'zuko@example.com'
-        )
-        expect(revisionsInTimeOrder[4].submitInfo?.updatedReason).toBe(
+        expect(submissionsInTimeOrder[4].rateRevisions).toHaveLength(2)
+        // expect(submissionsInTimeOrder[4].unlockInfo?.updatedReason).toBe(
```

Comment:
Can you add a comment for why these assertions are commented out?

Reply:
I commeneted it out because subsissionpackages don't have unlock info, but it turns out these tests are all skipped anyway so I'm going to fix them